sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -255,6 +254,29 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite {
     }
   }
 
+  test("SPARK-29911: Uncache cached tables when session closed") {
```

Comment:
Could we move this test to a new test suite because it is used to test these files:
https://github.com/apache/spark/tree/master/sql/core/src/test/resources/sql-tests

Reply:
Ah, that's a good point, I missed that. @LantaoJin what about a follow up for these two small items?

Reply:
Sure. I will create a follow up issue